tests/src/test/scala/cats/tests/NonEmptyListSuite.scala

```diff
@@ -321,6 +321,18 @@ class NonEmptyListSuite extends NonEmptyCollectionSuite[List, NonEmptyList, NonE
     }
   }
 
+  test("NonEmptyList#min is consistent with List#min") {
+    forAll { (nel: NonEmptyList[Int]) =>
+      assert(nel.min === nel.toList.min)
+    }
+  }
+
+  test("NonEmptyList#max is consistent with List#max") {
+    forAll { (nel: NonEmptyList[Int]) =>
+      assert(nel.max === nel.toList.max)
+    }
+  }
+
```

Comment:
After @posco's suggestions, which I agree with, these properties assume `Order` and `Ordering` are consistent.  That holds for `Int`, and I can't think of a counterexample.  But that assumption might be worth a comment.

Vaguely related discussion: #3936.